### PR TITLE
Extend ipc_level_zero.c example

### PR DESCRIPTION
**Pull request by vinser52**
### Description

- Added `level_zero_fill` and `level_zero_copy` functions to the `utils_level_zero.h` to be used in the L0 examaples.
- Extended `ipc_level_zero.c` example to fill the initial buffer with some pattern and verify that mapped buffer contains the same pattern.

### Checklist
- [x] Code compiles without errors locally
- [x] All tests pass locally
- [x] CI workflows execute properly

### CODE REVIEW

**Review by bratpiorka — APPROVED** [review]

File: examples/basic/utils_level_zero.h [hunk 1]
```diff
@@ -217,6 +217,179 @@ static inline int find_gpu_device(ze_driver_handle_t driver,
...
+        ZE_STRUCTURE_TYPE_COMMAND_LIST_DESC, 0, 0,
+        ZE_COMMAND_LIST_FLAG_RELAXED_ORDERING};
+
+    ze_command_queue_handle_t hCommandQueue;
+    ze_result_t ze_result = zeCommandQueueCreate(
+        context, device, &commandQueueDesc, &hCommandQueue);
+    if (ze_result != ZE_RESULT_SUCCESS) {
+        fprintf(stderr, "zeCommandQueueCreate() failed!\n");
+        ret = -1;
+        goto exit;
```
Comment (ldorau) [inline examples/basic/utils_level_zero.h:244]: Just `return -1`. The `exit:` label is not needed.

File: examples/basic/utils_level_zero.h [hunk 2]
```diff
@@ -217,6 +217,179 @@ static inline int find_gpu_device(ze_driver_handle_t driver,
...
```
Comment (ldorau) [inline examples/basic/utils_level_zero.h:330]: Just `return -1`. The `exit:` label is not needed.

**Review by ldorau — APPROVED** [review]
LGTM except of 2 minor issues.

File: examples/basic/utils_level_zero.h (hunk 2 above)
Comment (vinser52) [reply examples/basic/utils_level_zero.h:330]: Done

**Review by kswiecicki — APPROVED** [review]

**Pull request merged**